keyvault/client/keys/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Release History
 
 ## 4.0.0-preview.1 (2019-06-28)
-For release notes and more information please visit 
-https://aka.ms/azure-sdk-preview1-java
+Version 4.0.0-preview.1 is a preview of our efforts in creating a client library that is developer-friendly, idiomatic to the Java ecosystem, and as consistent across different languages and platforms as possible. The principles that guide our efforts can be found in the [Azure SDK Design Guidelines for
+.Java](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html).
+
+For release notes and more information please visit https://aka.ms/azure-sdk-preview1-java
+
+This library is not a direct replacement for `azure-keyvault`. Applications using that library would require code changes to use `azure-keyvault-keys`.
```

Comment:
"replacement for `azure-keyvault`" - should you use a more specific name that 'azure-keyvault' (because this relates to keys specifically) or a link to the repo or somewhere?

Reply:
added repo link and mentioned about keys/secret specific operations.